tck-sectionid-renaming.xsl

```diff
@@ -0,0 +1,117 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ JBoss, Home of Professional Open Source
+  ~ Copyright 2012, Red Hat, Inc. and/or its affiliates, and individual contributors
```

Comment:
Created https://hibernate.atlassian.net/browse/BVAL-584 so I don't forget about it.